src/server/bin/runner.ml

```diff
@@ -0,0 +1,23 @@
+(*
```

Comment:
Let's have this file along with the other runners in `src/runners` (unless there's a cyclic dependence caused, in which case it's fine).